.vscode/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"ms-vscode.cpptools",
```

Comment:
So, if I leave this extension enabled, it tries to parse *thousands* of files, and bombs. I had to tweak my local config to only include `app/` subdirectory, and then that worked. Perhaps I have way too many west/zephyr modules in my crufty local checkout?

Anyone else encountered issues w/ this?

Reply:
The C/C++ extension? I haven't had that problem, possibly because I have a c_cpp_properties.json set up. That said, I also use it for a project at work which is far larger than ZMK without issues.